matchypatchy/gui/display_media.py

```diff
@@ -348,6 +384,14 @@ def delete(self):
                         id = int(self.media_table.data_filtered.at[row, "id"])
                     self.mpDB.delete('media', f'id={id}')
                 del dialog
+                # Clear selection and update UI
+                self.media_table.table.clearSelection()
+                self.update_buttons()
+                # Reload updated data and thumbnails
+                data_available = self.load_table()
+                if data_available:
```

Comment:
you shouldn't need to reload thumbnails, but you might be working from an old version